gds_write/def2stream.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python3
+
+import klayout.db as db
```

Comment:
need that you're able to use the klayout python module rather than the full executable!

Reply:
I'm not sure if I fully understand you here. Could you please explain what do you mean? Is there anything to change regarding that import or KLayout Python Module in general?

Reply:
s/need/neat/, sorry for the confusion :) I'm glad you're using the python module here.